staging/README.md

```diff
@@ -0,0 +1,62 @@
+# Introduction
+
+Welcome to the staging tree!
+This tree contains experimental code that does not (yet) meet the quality
+requirements of RIOT.
+
+**WARNING**
+
+Do not depend on anything within this subfolder!  Everything in here might be
+broken or break or be changed or removed at any time, and everything is
+completely unsupported! API's from this tree might change without notice.
```

Comment:
"completely unsupported by the RIOT maintainers"? What I'm trying to say is that code could be supported by somebody, but it's not necessarily kept up to date by maintainers or contributors (for example when an API changes).

Reply:
Hm. I wanted to express that the code is experimental and the maintainers shouldn't be bothered with "why doesn't this work?" questions... Any suggestion?

Reply:
How about: it is not officially supported by RIOT maintainers 
(but may be supported by one or more individuals)